compiler/resolution/functionResolution.cpp

```diff
@@ -8307,6 +8307,7 @@ static void resolveSupportForModuleDeinits() {
 
 static void resolveExports() {
   std::vector<FnSymbol*> exps;
+  bool isLibrary = (fLibraryCompile || fMultiLocaleInterop);
```

Comment:
I don't know these variables well, but would've guessed that this would need to include `fLibraryFortran` and `fLibraryPython` as well?  Or (checking) is it the case that `fLibraryCompile` is flipped when any `--library` flag is used and that the lack of any other library flags implies that it's a C compile?  Looks like maybe that is the case (?).

Reply:
All of the various `--library` flags for single locale also set `fLibraryCompile` as well.

Reply:
Not related to this PR, but since we're talking about it: I believe most of the f* variables correspond pretty directly 1:1 to a given command-line flag, which is why I assumed fLibraryCompile only meant the `--library` flag when browsing the source the other day.  It seems like it would be more consistent to have a `isLibraryCompile()` routine that checked whether any of a number of flags had been thrown than to have various flags other than `--library` set this one.  And/or to have an fLibraryC variable that corresponds to `--library` (which should probably be renamed `--library-c` about now) so that it was more symmetric.  (I'd probably do both of these things, but again, not in this PR, just as housecleaning).

Reply:
These are my sentiments as well.

Reply:
That's fair.  I didn't look at precedent when extending the use of fLibraryCompile - I was approaching it from a minimalistic change attitude at the time.